apps/src/lab2/types.ts

```diff
@@ -17,6 +17,12 @@ import {lab2EntryPoints} from '../../lab2EntryPoints';
 
 export {Theme};
 
+/// ------ USER APP OPTIONS ------ ///
+
+export interface UserAppOptions {
```

Comment:
nit: maybe `PartialUserAppOptions`, along with a comment that we're only defining fields that we strictly need? Parallels `PartialAppOptions` in `utils.ts`